HISTORY.rst

```diff
@@ -28,6 +28,8 @@ Unreleased
 
 **Libraries**
 
+* Update marabunta to have fixed marabunta_serie
+
```

Comment:
Can you also announce the breaking changes? Like in https://github.com/camptocamp/docker-odoo-project/pull/102/files#diff-d9ae695c8119e58c2ec435b6deae2d53